addons/hr_recruitment_survey/models/survey_survey.py

```diff
@@ -0,0 +1,23 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class SurveySurvey(models.Model):
+    _inherit = 'survey.survey'
+
+    survey_type = fields.Selection(selection_add=[('recruitment', 'Recruitment')], ondelete={'recruitment': 'set default'})
+    job_ids = fields.One2many("hr.job", "survey_id")
```

Comment:
Ah it does not cost much to give the full model name ...
```suggestion
    hr_job_ids = fields.One2many("hr.job", "survey_id", string="Used for")
```